Assets/Scripts/UI/Canvas/StateMachine/UIStateEvent.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine.Events;
+
+namespace UI.Canvas.StateMachine
+{
+    [System.Serializable]
+    public class UIStateEvent : UnityEvent<StateChange>
+    {
+    }
```

Comment:
```suggestion
    [System.Serializable]
    public sealed class UIStateEvent : UnityEvent<StateChange> { }
```

Ik werdt erg verward toen ik dit voor het eerst zag, daarom een summary toevoegen is wel handig. [Voorbeeld](https://github.com/Team-Swamp/IceBites/blob/develop/Assets/Scripts/Framework/Cooking/HeldItem.cs)!